app/src/main/res/values-ru/strings.xml

```diff
@@ -158,10 +158,10 @@
     <string name="message_file_already_exists">Такой файл уже существует</string>
     <string name="message_file_not_found">Файл не найден</string>
     <string name="message_out_of_memory">Out of memory</string>
-    <string name="message_archive_unsupported">The archive are unsupported.</string>
-    <string name="message_zip_file_invalided">Zip file is invalided</string>
-    <string name="message_zip_file_split">Zip file is split</string>
-    <string name="message_zip_file_encrypted">Zip file is encrypted</string>
+    <string name="message_unsupported_archive">The archive are unsupported.</string>
+    <string name="message_invalid_archive">Zip file is invalided</string>
+    <string name="message_split_archive">Zip file is split</string>
+    <string name="message_encrypted_archive">Zip file is encrypted</string>
```

Comment:
Request translate to Russian

Reply:
Russian translation:
```suggestion
    <string name="message_unsupported_archive">Неподдерживаемый архив</string>
    <string name="message_invalid_archive">Архив поврежден</string>
    <string name="message_split_archive">Архив разделен</string>
    <string name="message_encrypted_archive">Архив зашифрован</string>
```
Also I would change English translation to:
```xml
<string name="message_unsupported_archive">Unsupported archive</string>
<string name="message_invalid_archive">Invalid archive</string>
<string name="message_split_archive">Archive is split</string>
<string name="message_encrypted_archive">Archive is encrypted</string>

Reply:
Agree, my English is poor,